src/widgets/forms/forms.component.tsx

```diff
@@ -68,6 +73,20 @@ const Forms: React.FC<any> = () => {
     setCompletedForms(completedForms);
   }, [allForms, encounters]);
 
+  React.useEffect(() => {
+    const filledForms = allForms.map(form => {
+      let checkedForm: Form;
+      completedForms.map(completeForm => {
+        if (completeForm.uuid === form.uuid) {
+          completeForm.checked = true;
+          checkedForm = completeForm;
+        }
+      });
+      return checkedForm ? checkedForm : form;
+    });
+    setFilledForms(filledForms);
+  }, [allForms, completedForms]);
```

Comment:
Wait, how is `filledForms` different from `completedForms`?

Reply:
`completedForms` are forms a user has filled for a patient, these forms are displayed on the second tab `Complete`.
`filledForms` are basically all forms but have the `Form.completed` property updated to display it as already completed under the `All` tab on the content switcher

Reply:
Why not just add the `completed` property to `forms` and get rid of `filledForms` entirely?

And then why not get rid of the `completedForms` state variable and just do `const completedForms = forms.filter(f => f.completed)`?